### PR TITLE
Add alert in promscale

### PR DESCRIPTION
**Pull request by ramonguiu**
# Description

Continue moving docs from Promscale repo to docs.timescale.com

This PR covers alerting.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

### CODE REVIEW

**Comment by Loquacity** [comment]
Needs editing

**Comment by Loquacity** [comment]
@ramonguiu @VineethReddy02 Ready for your review.

**Review by charislam — APPROVED** [review]

File: promscale/alert.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+To configure alerts in Promscale, use the alerting capabilities provided by
+Prometheus.
+
+Prometheus alerting rules are used to trigger alerts based on the violation of
+conditions. Alerts can be configured to send to external services such as Slack
+or email. Alerting rules are written in a YAML file and specified in the
+Prometheus configuration file. These conditional rules are evaluated by
+Prometheus, not by Promscale.
+
+Set up alerting by adding creating a YAML file containing configuration like
```
Comment (ramonguiu) [inline promscale/alert.md:11]: "adding creating" -> "creating" ?

**Review by ramonguiu — COMMENTED** [review]
Looks good. Added a small comment.

File: promscale/alert.md (hunk 1 above)
Comment (Loquacity) [reply promscale/alert.md:11]:
```suggestion
Set up alerting by creating a YAML file containing configuration like
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Comment by ramonguiu** [comment]
@Loquacity are there any additional reviews or changes required? It seems this one is ready to go.

**Review by VineethReddy02 — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

**Pull request merged**